# Data Science

With the support of supplemental funding, insitutions hosting the National Library of Medicine's [Biomedical Informatics and Data Science Ressearch Training Programs](https://www.nlm.nih.gov/ep/GrantTrainInstitute.html) developed data science training materials, with the express goal of providing these materials as shareable resources for the educational community. 

Links to these resources are provided below, grouped by institution.

---
# [University of Pittsburgh](https://www.dbmi.pitt.edu)

*Note*: These resources can be also be found at [datascience.dbmi.pitt.edu](https://datascience.dbmi.pitt.edu) 

## Social Media Data Science

[![DOI](https://zenodo.org/badge/147564959.svg)](https://zenodo.org/badge/latestdoi/147564959)

[GitHub Repository](https://github.com/dbmi-pitt/SocialMediaDataScience)

Social media can be a useful source of information regarding public discussion of health issues.  Using this data requires informatics strategies for retrieving, storing, annotating, interpreting, and classifying this data. 

This module challenges users to explore Twitter data, using the Python programming language, Twitter application programming interfaces, and libraries for natural language processing and machine learning.  Students will gain familiarity with the use of these tools to retrieve data, annotate data, conduct basic natural language processing tasks,  and build and evaluate simple classifier.

Students should have some background in Python programming. A Twitter developer account is required. Instructions on acquiring and configuring Twitter accounts are included.

# Differential expression analysis of RNA-Seq data from TCGA

[![DOI](https://zenodo.org/badge/170748515.svg)](https://zenodo.org/badge/latestdoi/170748515)

[GitHub Repository]

The examination of gene expression patterns in breast cancer samples can provide valuable insights into genetic activity underlying cancers and distinguishing between cancer subtypes. Although the public availability of gene expression data sets through resources such as the Cancer Genome Atlas (TCGA) provides opportunities for exploring and analyzing these datasets, asking meaningful questions often requires the examination of multiple data sets from various sources, manipulation of the data in preparation for analysis, and use of research tools for conducting analyses.

This project explores differential analysis of RNA-Seq data from TCGA, using the R statistical environment and Bioconductor tools, primarily the DESeq2 differential gene analysis package.
